kairon/shared/chat/processor.py

```
@@ -235,3 +239,71 @@
+        """
+        Uploads the file to BSP and deletes the temporary local file.
+        """
+        from ..channels.whatsapp.bsp.factory import BusinessServiceProviderFactory
+        media_id = None
+        try:
+            channel_config = ChatDataProcessor.get_channel_config(channel, bot)
+            bsp_type = channel_config.get("config").get("bsp_type", "whatsapp")
+            media_id = await (BusinessServiceProviderFactory.get_instance(bsp_type).upload_media_file(bot,
+                                channel_config, user, file_info.filename, file_info.content_type, file_info.size))
+            logger.info(f"Media uploaded successfully: {media_id}")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)